### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by deniodev**
- [ ] Add unit tests for all the methods in all the classes.

### CODE REVIEW

File: spec/person_spec.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      new_student = Person.new(13, name, parent_permission: true)
+      expect(new_student.can_use_services?).to eq true
+    end
+
+    it 'should return true when age is less than 18 and has their parent\'s permission' do
+      new_student = Person.new(17, name, parent_permission: true)
+      expect(new_student.can_use_services?).to eq true
+    end
+  end
+end
```
Comment (mckent05) [inline spec/person_spec.rb]: - Good job done so far, but according to the project requirements, you are to test all the methods in all classes. Please test the `add_rental` method in your Person class 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @deniodev, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- God job adding tests to your application 👏 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ragangithub** [comment]
- Added test for add_rental method in  Person class

**Review by OLIPLICHE — APPROVED** [review]
Hi @deniodev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Wrote unit tests for all the methods in all the classes (✅)) 👏
- All test has passed 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**